neuro_data/static_images/dataset_config.py

```diff
@@ -0,0 +1,533 @@
+            dynamic_scan = (
+                DvScanInfo
+                & {
+                    "animal_id": key["animal_id"],
+                    "session": key["dynamic_session"],
+                    "scan_idx": key["dynamic_scan_idx"],
+                }
+            ).fetch1("KEY")
+            static_scan = (
+                StaticScan()
+                & {
+                    "animal_id": key["animal_id"],
+                    "session": key["static_session"],
+                    "scan_idx": key["static_scan_idx"],
+                }
+            ).fetch1("KEY")
+            log.info("Fecthing images")
+            trial_idx, condition_hashes, images, types = (
+                InputConfig().part_table(key).input(static_scan)
+            )
+            log.info("Fetching responses")
+            responses = (DvScanInfo & key).responses(
+                dynamic_scan,
+                trial_idx=trial_idx,
+                condition_hashes=condition_hashes,
+            )
+            dynamic_unit_keys = (DvScanInfo & key).unit_keys(
+                dynamic_scan,
+            )
+            log.info("Fecthing tiers")
+            tiers = TierConfig().part_table(key).tier(static_scan, condition_hashes)
+            log.info("Fecthing layer information")
+            layer = LayerConfig().part_table(key).layer(dynamic_unit_keys)
+            log.info("Fecthing area information")
+            area = AreaConfig().part_table(key).area(dynamic_unit_keys)
+            log.info("Computing stats")
+            statistics = (
+                StatsConfig()
+                .part_table(key)
+                .stats(
+                    dynamic_scan, condition_hashes, trial_idx, images, responses, tiers
+                )
+            )
+            neurons = dict(
+                unit_ids=np.array([k["unit_id"] for k in dynamic_unit_keys]).astype(
+                    np.uint16
+                ),
+                animal_ids=np.array([k["animal_id"] for k in dynamic_unit_keys]).astype(
+                    np.uint16
+                ),
+                sessions=np.array([k["session"] for k in dynamic_unit_keys]).astype(
+                    np.uint8
+                ),
+                scan_idx=np.array([k["scan_idx"] for k in dynamic_unit_keys]).astype(
+                    np.uint8
+                ),
+                layer=layer.astype("S"),
+                area=area.astype("S"),
+            )
+            return dict(
+                images=images,
+                responses=responses,
+                types=types.astype("S"),
+                condition_hashes=condition_hashes.astype("S"),
+                trial_idx=trial_idx.astype(np.uint32),
+                neurons=neurons,
+                tiers=tiers.astype("S"),
+                statistics=statistics,
+            )
+
+
+@schema
+class DatasetInputResponse(dj.Computed):
+    # Inject datasets back to InputResponse table. Mutltiple datasets with the same
```

Comment:
may want to note that only insert back to the master table of InputResponse and the part tables would be empty

Reply:
good point, will do